scripts/check_code_style.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+check_source_naming() {
```

Comment:
WDYT of doing this via pre-commit?  This way you'll find out automatically before you commit.

You'd just use this "language" and match bad files: https://pre-commit.com/#fail

Reply:
Great idea, I'm gonna resume this PR after https://github.com/RespiraWorks/VentilatorSoftware/pull/131 lands.

Reply:
It has landed!